pkg/pillar/cmd/zedrouter/metadata.go

```diff
@@ -31,6 +31,10 @@ const SignerMaxSize = 65535
 // DiagMaxSize is the max returned size for diag
 const DiagMaxSize = 65535
 
+// PatchEnvelopeHandler is route used for patch envelopes
+// it is used in URL composing of patch envelopes
+const PatchEnvelopeHandler = "/eve/v1/patch/"
```

Comment:
What about `PatchEnvelopeURLPath` instead? In this file, handler is used to refer to the function handling the method.

Reply:
Makes sense